app/models/commit_monitor_branch.rb

```diff
@@ -31,4 +31,19 @@ def last_commit_uri
   def pr_number
     MiqToolsServices::MiniGit.pr_number(name) if pull_request?
   end
+
+  def write_github_comment(message, header, continuation_header = nil, replace = true)
+    return unless pull_request?
+
+    message_builder = MiqToolsServices::Github::MessageBuilder.new(header + message, continuation_header)
+
+    repo.with_github_service do |github|
+      @github = github
+      # TODO: Refactor the common "delete prior tagged issues" into miq_tools_services
+      # github.delete_issue_comments(pr_number, header) if replace
+      github.create_issue_comments(pr_number, message_builder.comments)
+    end
+
+    logger.info("#{self.class.name}##{__method__} Wrote comment with header: #{header} on PR: #{name}")
```

Comment:
Would be better to have the logging before the execution.  That way if there's a failure we can trace back to what it was trying to do.